docs/_layouts/default.html

```diff
@@ -10,7 +10,7 @@
   <div id="header">
     <a href="/" alt="NSubstitute">
       <img src="/images/nsubstitute-100x100.png" style="float: left; margin-right: 15px; width: 100px; height: 100px;" />
-      <h1 style="display: inline"> NSubstitute </h1> <br />
+      <h1 id="copy-disabled" style="display: inline"> NSubstitute </h1> <br />
```

Comment:
I added this because during implementation I noticed that the tooltip is also shown for the overall header. 

![image](https://github.com/nsubstitute/NSubstitute/assets/44709506/0fc22960-1fd0-4f00-b907-4a8178a3b2f5)


This looked weird to me to I fixed it.

Reply:
Isn't a CSS class a beter idea instead of a fixed id?

Reply:
done